<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Session</class>
 <widget class="QWidget" name="F_Session">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>440</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Label_Bienvenue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bienvenue Bidule !</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Train your Brain !</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_Stats">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Mes Statistiques</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_Supprimer_Session">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Supprimer Session</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_Calculate">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Calculate!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_Memorize">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Memorize!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_QuickShape">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>QuickShape!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_Retour_Accueil">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;&lt; Retour Acceuil</string>
   </property>
  </widget>
  <widget class="QLabel" name="Derniere_Connexion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Derni√®re Connexion :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
